elys/versions.json

```diff
@@ -2,6 +2,106 @@
   "$schema": "../versions.schema.json",
   "chain_name": "elys",
   "versions": [
+    {
```

Comment:
Good to include this info, but it's in the wrong order. The latest version should be at the end of the array, not the start

Reply:
@JeremyParish69 thanks for the heads up I just updated the order as suggested please review again thanks